app/assets/javascripts/jquery.slick.js

```diff
@@ -2017,46 +2065,45 @@
         _.$slideTrack.css(positionProps);
       }
     }
-
   };
 
   Slick.prototype.setDimensions = function() {
-
     var _ = this;
+    var $firstSlide = _.$slides.first();
+    var MAGIC_CONSTANT = 5000;
```

Comment:
Can use a single `var` declaration to make it more consistent with the style of the [existing code](https://github.com/bespokepost/jquery-slick-rails/blob/546748ea7df856268e28a1500bd6e8633e1aa160/app/assets/javascripts/jquery.slick.js#L735-L737).

Reply:
https://github.com/bespokepost/jquery-slick-rails/pull/1/commits/9e7ddf82a0e7d9fc9b17f9e92ca54881d5029e74